lib/semantic_logger/formatters/json_with_api_metadata.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module SemanticLogger
+  module Formatters
+    class JsonWithApiMetadata < Raw
+      def call(log, logger)
+        super(log, logger).merge(api_metrics).to_json
+      end
+
+    private
+
+      def api_metrics
+        return {} unless payload
+
+        {
+          params: payload[:params],
```

Comment:
maybe we can add the same rule here around not including `exceptions = %w[controller action format id]` similar to the old code: `payload[:params].except(*exceptions)`

Reply:
This would affect all the logs though, aren't those things important elsewhere? 

Maybe leave them in for now and if things become messy exclude them based on route later?